content/documentation/pages/6-recipes/5-kinesis/1-simple-producer-consumer.md

```diff
@@ -0,0 +1,170 @@
+---
+path: 'recipes/kinesis/simple-producer-consumer/'
+title: 'Amazon Kinesis Binder'
+description: 'A sample of Spring Cloud Stream + Amazon Kinesis Binder in action'
+---
+
+# Amazon Kinesis Sample
+
+The Spring team currently maintains the Spring Cloud Stream Kinesis binder with the help of community contributions.
+You can read more about the binder implementation at [spring-cloud/spring-cloud-stream-binder-aws-kinesis](https://github.com/spring-cloud/spring-cloud-stream-binder-aws-kinesis).
+
+In this walk-through, though, we will review a simple use-case to showcase how the Kinesis binder can be used with Spring Cloud Stream.
+
+## Prerequisite
+
+The only requirement for the demonstration is the "Access Key", "Secret Key", and "Region" credentials, which can be gathered from your AWS account.
```

Comment:
BTW, when we run our app in AWS EC2, we don't need credential properties, as well as `region`.
There are determined automaticaly from the AWS environment. 
Might be worth to mention